.github/workflows/flow-test.yml

```diff
@@ -87,6 +87,10 @@ on:
         description: 'Fail fast on test failures'
         type: boolean
         default: true
+      intel:
+        description: 'Run Intel tests on self-hosted EC2 runner'
+        type: boolean
+        default: true
```

Comment:
Consider making `intel` a new platform option to choose, instead of a checkbox (it doesn’t make sense to pick alpine+ARM, for example, and then get both this and an intel jobs)